transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -274,6 +274,28 @@ public void recordNextCallTime(TransportContext transportContext, long timestamp
         });
   }
 
+  @Override
+  public Iterable<TransportContext> loadActiveContexts() {
+    return inTransaction(
```

Comment:
Thinking out loud:
What guarantees for newly updating clients that this query is run after the client migrates to V2 schema ? If it were to run before, it may not find the extras column ?